src/main/java/step3/domain/Participants.java

```diff
@@ -0,0 +1,27 @@
+package step3.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Participants {
+    private List<Participant> participants;
+
+    private Participants(List<Participant> participants) {
+        this.participants = participants;
+    }
+
+    public static Participants of(List<String> participants) {
+        return new Participants(toParticipant(participants));
+    }
+
+    private static List<Participant> toParticipant(List<String> participants) {
+        return IntStream.range(0, participants.size())
+                .mapToObj(i -> Participant.of(Name.of(participants.get(i)), ParticipantPosition.of(i)))
```

Comment:
```suggestion
                .mapToObj(i -> Participant.of(participants.get(i), i))
```

외부에서 Name와 ParticipantPosition를 알지 못하면 Participant를 생성할 수 없겠네요!
일급 컬렉션과 원시값을 포장하는 것은 좋지만, 모든 정보를 외부에서 알아야한다면 해당 코드를 사용하는 클라이언트 입장에선 매우 불편해질 것 같아요 🤔 
원시값을 포장하며 얻는 이점과, 클라이언트의 편의성을 둘 다 높이는 방법이 뭐가 있을지 고민해보아요 😄